php/recipes/module-poppler-pdf.rb

```diff
@@ -12,6 +12,15 @@
   package package_name
 end
 
+poppler_install_dir = '/opt/easybib/lib/php/extensions/no-debug-non-zts-20131226'
+Chef::Log.debug("Copying #{poppler_install_dir}/poppler.so to PHP extension dir")
+execute "move-poppler-so" do
+  cwd poppler_install_dir
+  command 'cp poppler.so `/usr/bin/php5.6 -r "echo ini_get(\"extension_dir\");"`/poppler.so'
+  ignore_failure true
+  only_if do ::File.exists?("#{poppler_install_dir}/poppler.so") end
+end
+
```

Comment:
This is a non-breaking "quick fix" for the poppler not deploying to php5.6.  in that the easybib version of php would look into /opt/easybib/lib/php/extensions/no-debug-non-zts-20131226 for it's extensions directory.  This is more properly solved with a postinst script in the package however due to the number of PHP's and differences between them as well as complexity of rebuilding the poppler package outside of the CI environment and SPEED of implementation, this was solved here with toss-able code.